echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/googlechat/GoogleChatMessage.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2018 Netflix, Inc.
```

Comment:
s/Netflix/Google

Reply:
Seems like all the license headers have Netflix, should be Google.

Reply:
Thanks. All fixed